Assets/MRTK/Examples/Experimental/Elastic/Scripts/WidgetElasticDemo.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Microsoft.MixedReality.Toolkit.Experimental.Physics;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Experimental.Demos
+{
+    /// <summary>
+    /// Demonstration script to show how <see cref="ElasticSystem{T}"/> can be used to drive
+    /// a satisfying UI inflation/deflation effect of an example UI widget.
+    /// </summary>
+    public class WidgetElasticDemo : MonoBehaviour
```

Comment:
This script/class is purely example code for developers to see how they can leverage `ElasticSystem` in a "raw" way to drive UI interactions directly.